config.go

```diff
@@ -108,8 +108,10 @@ type Config struct {
 	//
 	// NOTE: when not using nsqlookupd, LookupdPollInterval represents the duration of time between
 	// reconnection attempts
-	LookupdPollInterval time.Duration `opt:"lookupd_poll_interval" min:"10ms" max:"5m" default:"60s"`
-	LookupdPollJitter   float64       `opt:"lookupd_poll_jitter" min:"0" max:"1" default:"0.3"`
+	LookupdPollInterval      time.Duration `opt:"lookupd_poll_interval" min:"10ms" max:"5m" default:"60s"`
+	LookupdPollJitter        float64       `opt:"lookupd_poll_jitter" min:"0" max:"1" default:"0.3"`
+	LookupdPollAliveDuration time.Duration `opt:"lookupd_pool_alive_duration" default:"60s"`
```

Comment:
naming idea ... Duration may be a bit redundant, and KeepAlive more clear

```suggestion
	LookupdPollKeepalive time.Duration `opt:"lookupd_pool_keepalive" default:"60s"`
```